README.rst

```diff
@@ -23,3 +23,52 @@ You can contact the Vumi development team in the following ways:
 
 Issues can be filed in the GitHub issue tracker. Please don't use the issue
 tracker for general support queries.
+
+Usage
+-----
+
+Voice transports may receive additional hints for how to handle outbound
+messages the ``voice`` section of ``helper_metadata``. The ``voice`` section
+may contain the following keys:
+
+:``speech_url``:
+    The URL where the voice file to be played can be found. If this field is
+    absent or ``None``, a text-to-speech engine will be used to generate a
+    suitable sound from the message ``content``, otherwise this voice file
+    will be played.
+
+    This can either be a string containing the URL, or a list of strings
+    containing URLs to sound files that should be joined to form the message.
+:``wait_for``:
+    Gather response characters until the given DTMF character is encountered.
+    Commonly either ``#`` or ``*``. If absent or ``None``, an inbound message
+    is sent as soon as a single DTMF character arrives.
+:``barge_in``:
+    A boolean value that if ``True``, stops the playback of the message when
+    a DTMF character arrives. This allows the response to the input to be
+    played immediately, rather than waiting for the first message to finish
+    playing before hearing the response message.
```

Comment:
There is no default value for `barge_in`, which makes the value `None`, which evaluates the same as `False` would. Maybe we should mention that the default value is `False`?